integration_tests/macros/e2e_tests/clear_tests.sql

```diff
@@ -1,10 +1,16 @@
 {% macro clear_tests() %}
     {% if execute %}
+        {% do drop_schema(elementary.target_database(), target.schema) %}
+
         {% set database_name, schema_name = elementary.get_package_database_and_schema('elementary') %}
         {% do drop_schema(database_name, schema_name) %}
-        {% set schema_name = schema_name ~ '__tests' %}
+
+        {% set tests_schema_name = elementary.get_config_var('tests_schema_name') %}
```

Comment:
maybe we should rename to 'elementary_tests_schema_name'?

Reply:
dbt provided the package namespaces so that you won't need to do `<my_package>_<config_var>`, and instead just:

```yml
my_package:
  config_var1: ...
  config_var2: ...
```

All of our other configuration keys don't include the `elementary` keyword and in my opinion, nor should this.